models/Asset/Folder.php

```diff
@@ -179,11 +180,18 @@ public function getPreviewImage(bool $force = false)
 
             if ($count && !$skipped) {
                 $localFile = File::getLocalTempFilePath('jpg');
-                imagejpeg($collage, $localFile, 60);
+                if (false === imagejpeg($collage, $localFile, 60)) {
+                    Logger::info('Creation of collage file of asset folder ' . $this->getRealFullPath() . ' is failed.');
 
-                if (filesize($localFile) > 0) {
-                    $storage->write($cacheFilePath, file_get_contents($localFile));
+                    return null;
                 }
+                $localFileContent = file_get_contents($localFile);
+                if (false === $localFileContent) {
+                    Logger::info('Creation of collage file of asset folder ' . $this->getRealFullPath() . ' is failed.');
```

Comment:
Suggesting to slightly change the message a little too, might help to distinguish whether was a problem with the converter itself or if the converted temp file might be empty/faulty, something like:

```suggestion
                    Logger::info('Generated collage file of asset folder ' . $this->getRealFullPath() . ' is broken or cannot be found.');
```